models/schema/userSchema.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+mongoose.set('useCreateIndex', true);
+
+const schema = mongoose.Schema({
```

Comment:
💡 I think the variable name could be more descriptive (`userSchema` ?), as there will be multiple schemas for collections in the DB, which could make it more difficult to debug.